### ISSUE TITLE
Request - add support for networkx 3.x.x

### ISSUE THREAD

**Issue by erivlis** [geronimo-iia/networkx-query#6]
Can you add support for networkx 3.x.x?
If not can you summarize what are the issues an/or blockers for that?

Whatever the case - Thanks for the great work!

**Comment by geronimo-iia** [comment]
I can start a branch on it and see what goes wrong :)

**Comment by geronimo-iia** [comment]
https://github.com/geronimo-iia/networkx-query/pull/7

**Comment by geronimo-iia** [comment]
Done with 2.0.0 version :)
